.vimrc

```diff
@@ -57,7 +65,7 @@ call plug#end()
 " Performance boosters
 :set nocursorline
 :set lazyredraw
-:set re=1
+:set re=0
```

Comment:
I should rever this change and find a way to make it work with jsx and tsx files